tests/src/environment/crash_listener.ts

```diff
@@ -1,12 +1,12 @@
 export function crashListener(
     pid: number,
     component: string,
-    logFile: string
+    logFile: string,
```

Comment:
Can we apply a configuration to get a result that does not change so many files?

Reply:
Will try!

Reply:
I tried with `"trailingCommas": "never"` but the diff shrinks only by a couple of lines because we are currently not consistent on this topic. I do think that `"trailingCommas": "onlyMultiLine"` - which is the default - is better though so I am gonna keep it I think!